feature-libs/product-configurator/rulebased/occ/variant/converters/occ-configurator-variant-price-normalizer.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from '@angular/core';
+import { Converter } from '@spartacus/core';
+import { ConfiguratorModelUtils } from '@spartacus/product-configurator/common';
+import { OccConfigurator } from '../variant-configurator-occ.models';
+import { Configurator } from './../../../core/model/configurator.model';
+
+@Injectable({ providedIn: 'root' })
+export class OccConfiguratorVariantPriceNormalizer
+  implements Converter<OccConfigurator.Prices, Configurator.Configuration> {
+  convert(
+    source: OccConfigurator.Prices,
+    target?: Configurator.Configuration
```

Comment:
Why are we converting to a configuration object which we do not use any further?

Reply:
The config object is only used to read the price summary and the price supplements